eth/beacon/helpers.py

```diff
@@ -402,6 +404,32 @@ def get_domain(fork_data: 'ForkData',
         slot,
     ) * 4294967296 + domain_type
 
+def verify_slashable_vote_data(state: BeaconState, vote_data: SlashableVoteData) -> bool:
```

Comment:
Since you're using `TYPE_CHECKING`, here should be:
```suggestion
def verify_slashable_vote_data(state: BeaconState, vote_data: 'SlashableVoteData') -> bool:
```